jupyterhub/objects.py

```diff
@@ -149,6 +149,15 @@ class Hub(Server):
 
     cookie_name = 'jupyter-hub-token'
 
+    pending_spawns = Integer(
```

Comment:
pending_spawn_count? spawn_pending_count? We already use `spawner._spawn_pending`, so perhaps `spawn_pending_count` to be consistent with that?